constantine-go/constantine.go

```diff
@@ -265,4 +265,585 @@ func (ctx EthKzgContext) VerifyBlobKzgProofBatchParallel(tp Threadpool, blobs []
 		return false, err
 	}
 	return true, nil
-}
\ No newline at end of file
+}
+
+// Ethereum BLS signatures
+// -----------------------------------------------------
```

Comment:
I think this needs a rebase on master as this should already be there from #383 

See https://github.com/mratsim/constantine/pull/383/files#diff-66f2a828703913aa04c16d50c2e86f4f3ffd7551cccee3a74a6ddafc00acad75R271